keps/sig-storage/1885-csi-migration-azurefile/README.md

```diff
@@ -40,6 +46,36 @@ used as as described in its parent KEP. For all other contents, please refer to
 
 Please refer to the [CSI Migration Production Readiness Review Questionnaire](https://github.com/kubernetes/enhancements/tree/master/keps/sig-storage/625-csi-migration#production-readiness-review-questionnaire).
 
+## Design Details
+### Test Plan
+
+[X] I/we understand the owners of the involved components may require updates to
+existing tests to make this code solid enough prior to committing the changes necessary
+to implement this enhancement.
+
+##### Prerequisite testing updates
+
+No additional tests are needed, rather the issue is orchestrating CSI driver
+deployment for prow jobs. This has been complicated by the cloud provider
+extraction work, which no longer permits cloud provider specific orchestration
+in the k/k repository. This means that it is not possible to run any test for
+AzureFile in k/k. All such tests have been moved to the [Azure File CSI Driver repo](https://github.com/kubernetes-sigs/azurefile-csi-driver).
+
+##### Unit tests
+
+N/A
```

Comment:
List this test: https://github.com/kubernetes/csi-translation-lib/blob/master/plugins/azure_file_test.go

Reply:
fixed.